crates/ika-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -509,7 +536,11 @@ impl DWalletMPCManager {
                     "received a message for an MPC session ID: `{:?}` which an event has not yet received for",
                     message.session_id
                 );
-                self.push_new_mpc_session(&message.session_id, None);
+                self.push_new_mpc_session(
```

Comment:
We said we want to limit now what sessions can we receive message for and mark validators that send messages for inexisting sessions as malicious